synapse/lib/types.py

```diff
@@ -552,6 +552,7 @@ def __init__(self):
         self.types = {}
         self.subtypes = []
 
+    def addDefaultTypes(self):
```

Comment:
potential alternative might be to move the statement `DataModel.__init__(self)` halfway down the Cortex ctor.